ldes-server-infra-mongo/mongock-changesets/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/mongock/changeset10/REAME.md

```diff
@@ -0,0 +1,18 @@
+# Mongock Changeset 10
+
+As part of the Isolation of the Retention module, the ldesmember table has been removed and replaced with member_properties.
+This new table does not persist all the previous properties.
+
+Following configuration is needed to apply this changeset.
+
+```
+mongock:
+  migration-scan-package:
+    - be.vlaanderen.informatievlaanderen.ldes.server.infra.mongo.mongock.changeset10
+```
+
+Consequences of Changeset:
+* renamed table ldesmember -> member_properties
```

Comment:
it's not really a rename though.. 
Since this table is renamed to ingest_ldesmember
Also this statement does not reflect what mongock does ;)